src/status_im2/contexts/wallet/events.cljs

```diff
@@ -321,3 +321,24 @@
 (rf/reg-event-fx :wallet/select-send-address
  (fn [{:keys [db]} [address]]
    {:db (assoc db :wallet/send-address address)}))
+
+(rf/reg-event-fx :wallet/get-address-details-success
+ (fn [{:keys [db]} [{:keys [hasActivity]}]]
+   {:db (assoc-in db
+         [:wallet :ui :watch-address-activity-state]
+         (if hasActivity :has-activity :no-activity))}))
```

Comment:
`hasActivity` is not converted to kebab case as we just need that one key and also we don't store the data (keys) in app-db.

Reply:
It's hard to answer, I'd prefer never having camelCase keys, but renaming just for a single key is too much.
maybe we could use an interceptor for wallet related events so we don't need to transform them after each request manually?

Another option would be to modify the `:json-rpc/call` fx, so we can add an optional parameter like, for example,
`:transform-keys true`, so if we add it, the response is always kebab-case. It'd be optional since other parts of the app are probably using camelCase sometimes.

Reply:
I really liked the second idea of introducing an additional key `:transform-keys?` to convert the camelCase to kebab-case. It's less prone to bugs and provides flexibility to be used by other (non-wallet) RPC calls. 

However, I'm concerned about the performance issues we face with `camel-snake-kebab` in communities data keys renaming https://github.com/status-im/status-mobile/issues/17282. This is good for a small amount of data. If the data is large such as in communities and chat/messages, it would take a lot of time to convert every key.

We can explore other possible solutions for conversation. 
Good idea! @ulisesmac